lib/filedrop.ts

```diff
@@ -106,13 +128,27 @@ export class FileDropElement extends HTMLElement {
     this.setAttribute('accept', val);
   }
 
+  get multiple() : string | null {
+    return this.getAttribute('multiple');
+  }
+
+  set multiple(val: string | null) {
+    this.setAttribute('multiple', val || '');
```

Comment:
This will set the `multiple` attribute even when I want to un-set it with `va = null`.

Reply:
You can't setAttribute null as it would set it to a string of 'null' - https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute